docs/platforms/dart/common/debug-symbols/index.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Debug Symbols
+description: "Learn about uploading debug symbols to enable symbolication of stack traces in your Dart applications."
+sidebar_order: 3
+---
+
+Debug symbols are essential for understanding stack traces in your Dart application when errors occur. Without debug symbols, stack traces from minified or obfuscated code can be difficult or impossible to interpret.
+
+## What Are Debug Symbols?
+
+Debug symbols provide the necessary information to convert program addresses back to function names, source file names, and line numbers. When you build a Dart application, especially with obfuscation or optimization enabled, this information is typically removed from the main application bundle to reduce size.
+
+In Dart applications, debug symbols can include:
+
+- **For Dart code**: Symbol files generated when using `--split-debug-info` and `--obfuscate` flags
+- **For iOS/macOS**: dSYM files
+- **For Android**: Proguard/R8 mapping files and symbols files
+- **For Flutter Web**: Source maps
+- **For Android NDK**: ELF DWARF debug files
```

Comment:
Can we remove this? we also explain what to upload in the `Upload Methods` below section as well